deadspace/code/corruption/corruption.dm

```diff
@@ -4,7 +4,7 @@
 #define IDLE 4
 
 // Amount of integrity lost and gained per second
-#define INTEGRITY_PER_SECOND 4
+#define INTEGRITY_PER_SECOND 3
```

Comment:
a 25% growth speed reduction is more than "slightly" but the 2.0 corruption moves at turbo speed compared to the 1.0 corruption so its fine imo